check-dnt.js

```diff
@@ -0,0 +1,12 @@
+function toggleStatus(enabled) {
+    var status = window.document.getElementById("dnt-" + enabled);
+    var waiting = window.document.getElementById("dnt-waiting");
+    waiting.classList.add("hidden");
+    status.classList.remove("hidden");
+}
+
+if (navigator.doNotTrack == "yes") {
```

Comment:
Testing locally i saw that "missing dnt signal" came up every time.

My browser returns an integer for navigator.doNotTrack

See https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack